src/redux/actions/auth/authActions.js

```diff
@@ -20,9 +20,13 @@ const loginAction = ({ username, password }) => (dispatch) => {
     return dispatch(login(response));
   }).catch((error) => {
     if (error.response) {
-      return dispatch(loginFailed(error.response.data.detail));
+      let errorMessage = error.response.data.detail;
+      if (errorMessage === 'Not found.' || errorMessage === 'Passwords did not match.') {
```

Comment:
I think the not found should not be used in this condition.

Reply:
@Duncanian, please elaborate more.